### PR TITLE
Fix redirect of `https://sustainability.eionet.europa.eu` to `https:/…

### PR DESCRIPTION
**Pull request by stevepiercy**
…/www.eea.europa.eu/en/topics/at-a-glance/sustainability/`

See https://github.com/plone/volto/pull/5941

### CODE REVIEW

**Review by ichim-david — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -284,7 +282,11 @@ The following websites have been built with Volto. You can find their complete s
 - [2023.ploneconf.org](https://github.com/plone/ploneconf.org/tree/2023) - Volto project for [Plone Conference 2023 site](https://2023.ploneconf.org)
 - [plone.org.br](https://github.com/plonegovbr/plone.org.br) - Volto project for the [Brazilian Plone Community](https://plone.org.br)
 - [nsw-design-system-plone6-kit](https://github.com/pretagov/nsw-design-system-plone6-kit) - NSW Design System Plone 6 Kit Volto project for [NSW.gov.au sites](https://digitalnsw.pretagov.com.au/)
+- [volto-eea-design-system](https://github.com/eea/volto-eea-design-system) - EEA Design System Plone 6 Kit Volto project for [European Environment Agency web sites](https://eea.github.io/volto-eea-design-system/)
```
Comment (avoinea) [inline README.md:285]: @stevepiercy Ops, we have a duplicate here :)

**Review by avoinea — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -284,7 +282,11 @@ The following websites have been built with Volto. You can find their complete s
...
 - [volto-centraalmuseum-theme](https://github.com/intk/volto-centraalmuseum-theme) - Volto project for the [Centraal Museum & Rietveld](https://www.centraalmuseum.nl/nl) made for [INTK](https://www.intk.com/en).
+- [volto-eea-design-system](https://github.com/eea/volto-eea-design-system) - EEA Design System Plone 6 Kit Volto project for [European Environment Agency web sites](https://eea.github.io/volto-eea-design-system/)
+- [volto-eea-website-theme](https://github.com/eea/volto-eea-website-theme) - EEA Plone 6 Volto Theme for [European Environment Agency web sites](https://www.eea.europa.eu/en)
```
Comment (avoinea) [inline README.md:288]: Also this one, is already in `Themes` section

**Review by avoinea — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (stevepiercy) [reply README.md]: Fixed in https://github.com/collective/awesome-volto/pull/20/commits/a7a033c74d866c7d0ec3ae2f8e3f948762ae0443

File: README.md (hunk 2 above)
Comment (stevepiercy) [reply README.md]: Fixed in https://github.com/collective/awesome-volto/pull/20/commits/a7a033c74d866c7d0ec3ae2f8e3f948762ae0443

**Review by avoinea — APPROVED** [review]

**Pull request merged**